packages/cozy-harvest-lib/test/components/KonnectorModal.spec.js

```diff
@@ -13,22 +13,51 @@ const findAccountMock = jest.fn().mockImplementation(() => ({
 }))
 
 describe('KonnectorModal', () => {
-  it('should pass trigger if konnector has triggers', async () => {
-    const mockKonnector = {
-      slug: 'mock',
-      name: 'Mock',
-      triggers: {
-        data: [{ id: 784, doctype: 'io.cozy.triggers' }]
-      }
+  const mockKonnector = {
+    slug: 'mock',
+    name: 'Mock',
+    triggers: {
+      data: [{ id: 784, doctype: 'io.cozy.triggers' }]
     }
-    const props = {
-      findAccount: findAccountMock,
-      dismissAction: jest.fn(),
-      onSuccess: jest.fn(),
-      konnector: mockKonnector,
-      t
+  }
+  const props = {
+    findAccount: findAccountMock,
+    dismissAction: jest.fn(),
+    createAction: jest.fn(),
+    onSuccess: jest.fn(),
+    konnector: mockKonnector,
+    t
+  }
+  const shallowOptions = {
+    context: {
+      client: {}
     }
-    const component = await shallow(<KonnectorModal {...props} />)
+  }
+
+  it('should show a spinner while loading', () => {
+    const component = shallow(<KonnectorModal {...props} />, shallowOptions)
+
+    const spinner = component
+      .dive()
+      .find('ModalContent')
+      .childAt(0)
+      .dive()
```

Comment:
Can't we do find(Spinner) directly ? Is it because the shallow render prevents us from doing that ?

Reply:
Yep, that's why!